packages/metal-dom/test/events.js

```diff
@@ -102,4 +109,86 @@ describe('Custom Events', function() {
 		assert.strictEqual(1, listener.callCount);
 		assert.strictEqual('animationend', listener.args[0][0].customType);
 	});
+
+	it('should listen to the keydown keyboard event by keyCode alias', function() {
+		var element = document.createElement('input');
+		dom.enterDocument(element);
+
+		var listener = sinon.stub();
+		var handle = dom.on(element, 'keydown:enter', listener);
+		dom.triggerEvent(element, 'keydown', {keyCode: KEYMAP.ENTER});
+		assert.strictEqual(1, listener.callCount);
+	});
+
+	it('should delegate parameterized keyboard event', function() {
+		var listener = sinon.stub();
+		var handle = dom.delegate(this.element4, 'keydown:enter', this.element5, listener);
+		dom.triggerEvent(this.element5, 'keydown', {keyCode: KEYMAP.ENTER});
+		assert.strictEqual(1, listener.callCount);
+	});
+
+	it('should listen to the keypress keyboard event by keyCode alias', function() {
+		var element = document.createElement('input');
+		dom.enterDocument(element);
+
+		var listener = sinon.stub();
+		var handle = dom.on(element, 'keypress:enter', listener);
+		dom.triggerEvent(element, 'keypress', {keyCode: KEYMAP.ENTER});
+		assert.strictEqual(1, listener.callCount);
+	});
+
+	it('should listen to the keyup keyboard event by keyCode alias', function() {
+		var element = document.createElement('input');
+		dom.enterDocument(element);
+
+		var listener = sinon.stub();
+		var handle = dom.on(element, 'keyup:enter', listener);
+		dom.triggerEvent(element, 'keyup', {keyCode: KEYMAP.ENTER});
+		assert.strictEqual(1, listener.callCount);
+	});
+
+	it('should listen to keyboard events by multiple key alias', function() {
+		var element = document.createElement('input');
+		element.setAttribute('type', 'text');
+		dom.enterDocument(element);
+
+		var listener = sinon.stub();
+		var handle = dom.on(element, 'keydown:enter,space,esc,up,down', listener);
+		dom.triggerEvent(element, 'keydown', {keyCode: KEYMAP.ENTER});
+		dom.triggerEvent(element, 'keydown', {keyCode: KEYMAP.SPACE});
+		dom.triggerEvent(element, 'keydown', {keyCode: KEYMAP.ESC});
+		dom.triggerEvent(element, 'keydown', {keyCode: KEYMAP.UP});
+		dom.triggerEvent(element, 'keydown', {keyCode: KEYMAP.DOWN});
+		assert.strictEqual(5, listener.callCount);
+	});
+
+	it('should not trigger the listener to inexistent key alias', function() {
+		var element = document.createElement('input');
+		dom.enterDocument(element);
+
+		var listener = sinon.stub();
+		var handle = dom.on(element, 'keyup:play', listener);
+		dom.triggerEvent(element, 'keyup', {keyCode: KEYMAP.SPACE});
+		assert.strictEqual(0, listener.callCount);
+	});
+
+	it('should not trigger the listener to an inexistent key alias', function() {
```

Comment:
The test description seems wrong here, it's the same description as the one in the previous test, but it's actually testing that it won't trigger a listener for the wrong key alias, not a nonexistent one.